Calculator.Tests/Calculator.Tests/LoginPageTests.cs

```diff
@@ -134,11 +134,33 @@ public void SpaceInLoginTest()
             Assert.AreEqual("Incorrect user name!", actual);
 
         }
-        
-       
+        [Test]
+        [Obsolete]
+        public void RemindPassBtnIsDisplayedTest()
+        {
+            new WebDriverWait(browser, TimeSpan.FromSeconds(10))
+            .Until(ExpectedConditions.VisibilityOfAllElementsLocatedBy(By.Id("remindBtn")));
+            IWebElement btn = browser.FindElement(By.Id("remindBtn"));
+            string actual = btn.Text;
+            Assert.AreEqual("Remind password", actual);
+
+        }
+        [Test]
+        public void LoginFieldName()
+        {
+            string LoginName = browser.FindElement(By.ClassName("user")).GetAttribute ("innerText");
```

Comment:
Please remove space after 'GetAttribute'

```suggestion
            string LoginName = browser.FindElement(By.ClassName("user")).GetAttribute("innerText");
```

Reply:
done